From 4a6cc02146a21498fc70d052ad3c5d4c0b1b597a Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 20 May 2020 12:19:21 +0300
Subject: [PATCH] [SW-0] gaudi: fix header license

Update header to match the license specification expectation and also
remove author, as all the team develops everything.

Change-Id: I2d612fa22ead4b0d6d11f0c3e435e4900a04ed59
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/common.h b/gaudi/common.h
index 8d132b5..6bae7a1 100644
--- a/gaudi/common.h
+++ b/gaudi/common.h
@@ -1,8 +1,7 @@
+/* SPDX-License-Identifier: GPL-2.0+ */
 /*
- * (C) Copyright 2019 HabanaLabs, Ltd.
- * Noam Camus <noamikong@gmail.com>
- *
- * SPDX-License-Identifier:     GPL-2.0+
+ * (C) Copyright 2019-2020 HabanaLabs, Ltd.
+ * All Rights Reserved.
  */
 
 #include <preboot.h>
